processor/batchprocessor/batch_processor.go

```diff
@@ -200,17 +201,17 @@ func (bp *batchProcessor) ConsumeLogs(_ context.Context, ld plog.Logs) error {
 }
 
 // newBatchTracesProcessor creates a new batch processor that batches traces by size or with timeout
-func newBatchTracesProcessor(set component.ProcessorCreateSettings, next consumer.Traces, cfg *Config, registry *featuregate.Registry) (*batchProcessor, error) {
+func newBatchTracesProcessor(set processor.CreateSettings, next consumer.Traces, cfg *Config, registry *featuregate.Registry) (*batchProcessor, error) {
```

Comment:
s/newBatchTracesProcessor/newBatchTraces?

Reply:
There already exists `newBatchTraces`, `newBatchMetrics`, `newBatchLogs` functions. Maybe this can be reworked later?

Reply:
sure.